hazelcast/connection.py

```diff
@@ -516,10 +531,7 @@ def _on_auth(self, response, connection):
             return self._handle_successful_auth(response, connection)
 
         if status == _AuthenticationStatus.CREDENTIALS_FAILED:
-            err = AuthenticationError(
-                "Authentication failed. The configured cluster name on "
-                "the client does not match the one configured in the cluster."
-            )
+            err = AuthenticationError("Authentication failed. Check cluster name and credentials.")
```

Comment:
Maybe make the error message better with something like this.

```suggestion
            err = AuthenticationError("Authentication failed. Either the configured cluster name on the client does not match with the one configured on the cluster or the credentials set in the client could not be authenticated.")
```
(Please make line shorter by line breaks if you are going to make this change)

Reply:
Shorter one conveys the same information unless I am missing anything.

Reply:
Yes, they are more or less the same but I feel like the longer one expresses the situation better.
The authentication failed -> Here are the causes
Of course, you can say that the shorter version might say the same thing.
I am resolving this, feel free to ignore this